env.example

```diff
@@ -0,0 +1,15 @@
+# this is the externally available public path
+# that connects the client dev server to the correct
+# host, in case you have to override it
+BASE_PATH=/
+API_BASE_PATH=/
+STATIC_DIRECTORY=/static
+PUBLIC=https://twodotoh.a.influxcloud.dev.local/
+UI_SHA=local_dev_mode
+CLOUD_URL=/auth
+
+# the gateway oidc provider plugin
```

Comment:
can get rid of all these comments, and change all references of `PORT_HTTPS` to `PORT`? It made sense in monitor-ci when it references the ingress port (which might be different from the service port), but not so much here when it's just the service